modules/social_features/social_core/social_core.module

```diff
@@ -1121,3 +1121,23 @@ function social_core_preprocess_status_messages(&$variables) {
     }
   }
 }
+
+/**
+ * Implements hook_menu_links_discovered_alter().
+ */
+function social_core_menu_links_discovered_alter(array &$links): void {
+
+  /** @var \Drupal\block_content\Entity\BlockContent $block */
+  $block = \Drupal::entityTypeManager()->getStorage('block_content')->loadByProperties(['type' => 'hero_call_to_action_block']);
+  $block = reset($block);
+
+  if ($block) {
```

Comment:
I think we need to make this a little more robust:
If a user follows the link, and decides to delete the block for whatever reason, the link is still there (cached?) but also, what if multiple `hero_call_to_action_blocks`
get added? Some could be added on a landingpage or dashboard, and don't represent the one added on the front-page. 

Maybe we can specify the Properties a bit better for that scenario.

![Screenshot 2022-02-10 at 12 35 36](https://user-images.githubusercontent.com/16667281/153401151-994974e4-395f-479b-ad03-0640096b3408.png)